### ISSUE TITLE
Sample MySQL Database required

### ISSUE THREAD

**Issue by ashishpatel1992** [informatici/survethi-monitoring-tool#34]
Hi @mwithi ,
Please add the sample database for testing purpose.

**Comment by mwithi** [comment]
Hi @ashishpatel1992,
the tool is meant to work, as at it's first stage, on a specific project in an hospital and we cannot disclose their DB for obvious reasons. To extract only some data for testing purpose may require time, meanwhile you can you use [tests endpoints](https://github.com/informatici/survethi-monitoring-tool#static-datasets-for-testing) I've prepared, which will use files in `datasource/` folder.

If something is missing I can add it (I've suspended testing for urgent deadlines)

Thanks!

**Comment by ashishpatel1992** [comment]
@mwithi You can provide Schema for the database. A dummy data can be populated afterwards

**Comment by mwithi** [comment]
it's quite useless the schema because you need several tables to be populated in order to have something working.

Anyway if you want to try... (rename it to .sql)

[step_00_db_structure.txt](https://github.com/informatici/survethi-monitoring-tool/files/5461434/step_00_db_structure.txt)

**Comment by ashishpatel1992** [comment]
Yes, I can understand the data is confidential. I found a good tool that might help in generating dummy data.
I will try this http://filldb.info/dummy

**Comment by mwithi** [comment]
What I'm saying is that you would end up with the data in `datasource/` folder... that is used by tests endpoints (you have just to replace the URL in the code).

To build up the whole DB and to fill it is cumbersome and worthless.

**Comment by ashishpatel1992** [comment]
Sure. I will try
